guide/designing-products/wallet-interoperability.md

```diff
@@ -232,11 +232,11 @@ Your application should support constructing and managing transaction files, als
 
 ---
 
-To kickstart your design work, take a look at [design resources]({{ '/guide/designing-products/design-resources/' | relative_url }}).
+Next, we cover how to onboard new users to a bitcoin daily spending wallet [onboarding]({{ '/guide/onboarding/' | relative_url }}).
```

Comment:
```suggestion
Next, we cover how to onboard new users to a bitcoin daily spending wallet [onboarding]({{ '/guide/onboarding/introduction/' | relative_url }}).
```

Reply:
This link showed up broken in `rake test` and returns 404 on deploy preview, it should match the button below it.